wiki/Ranking_Criteria/en.md

```diff
@@ -31,22 +31,17 @@ Glossary
+- **feat.**: featuring
 - **CV**: character voice, usually used to refer to the voice actor of a fictional character.
-- **Lenticular brackets**: Also known as Sumitsukikakko and written as `【】`.
-- **Corner brackets**: Also known as Kagikakko and written as `「」`.
-- **Square brackets**: `[]`
-- **asterisk**: `*`
-- **umlauts**: `ü`, `ö`, `ä` and `ß`
-- **TV Size**: A song where the particular version is used in a television program, web series, or direct to video series, such as an opening, ending, or insert song. Indie/doujin series are not included in this.
-
-Note: If a beatmapset's song contains matching sections to the TV size song, in the same order, and is roughly the same length as the official TV Size song, the beatmapset's edit will also count as TV Size
-
+- **Asterisk**: `*`
+- **Umlauts**: `ü`, `ö`, `ä` and `ß`
+- **Whitespace**: A visual spacing between characters, not always a literal space. If a character already has space in it, such as all of Japanese, that counts as whitespace.
+- **TV Size**: A song where the particular version is used in a television program, web series, or direct to video series, such as an opening, ending, or insert song. Fan-made works are not included in this.
 - **Modified Hepburn Romanisation**: Refer to [this page](https://en.wikipedia.org/wiki/Hepburn_romanization#Features_of_Hepburn_romanization) for information.
-- **Russian Romanisation**: Use `BGN/PCGN` system for Russian/Cyrillic. `Е` and `е` should be romanised as `ye` if it stands alone or after `a`, `e`, `ё`, `и`, `о`, `у`, `ы`, `э`, `ю`, `я`, `й`, `ъ`, `ь`. In other cases, it should be romanised as `e`. `ё` should be romanised to `yo`, however, use `o` if the character comes after `ж`, `ч`, `ш`, or `щ`. Ignore any other rules in the file provided, these are either irrelevant or wouldn't help in the game. If an artist uses a preferred romanisation, follow it regardless of this rule. For most of the other characters, refer to the [first page of this document](https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/526837/Romanization_system_for_Russian_31May2016.pdf).
-- **Character-by-character romanisation**: each Chinese character must be Romanised as a capitalised word and separated with a space.
+- **Russian Romanisation**: Use BGN/PCGN system for Russian/Cyrillic. Е and е should be romanised as `ye` if it stands alone or after `a`, `e`, `ё`, `и`, `о`, `у`, `ы`, `э`, `ю`, `я`, `й`, `ъ`, `ь`. In other cases, it should be romanised as `e`. `ё` should be romanised to `yo`, however, use `o` if the character comes after `ж`, `ч`, `ш`, or `щ`. Ignore any other rules in the file provided, these are either irrelevant or wouldn't help in the game. If an artist uses a preferred romanisation, follow it regardless of this rule. For most of the other characters, refer to the [first page of this document](https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/526837/Romanization_system_for_Russian_31May2016.pdf).
```

Comment:
doesn't seem like this belongs in the glossary?

Reply:
this is a method description so that the rule that mentions it doesnt become a clusterfuck - idk where else to put this

Reply:
in the Romanisation section under Metadata, probably